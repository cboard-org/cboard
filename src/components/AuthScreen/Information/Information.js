import React, { Fragment } from 'react';
import { FormattedMessage } from 'react-intl';
import Typography from 'material-ui/Typography';

import messages from './Information.messages';

const Information = () => (
  <Fragment>
    <Typography
      className="AuthScreen__heading"
      align="center"
      variant="display3"
    >
      <FormattedMessage {...messages.heading} />
    </Typography>
    <Typography className="AuthScreen__heading" align="center" variant="body1">
      <FormattedMessage {...messages.text} />
    </Typography>
  </Fragment>
);

export default Information;
