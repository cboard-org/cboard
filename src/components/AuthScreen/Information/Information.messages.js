import { defineMessages } from 'react-intl';

export default defineMessages({
  heading: {
    id: 'cboard.components.AuthScreenInformation.heading',
    defaultMessage: 'Cboard'
  },
  text: {
    id: 'cboard.components.AuthScreenInformation.text',
    defaultMessage: 'Sign up to sync your settings!'
  }
});
