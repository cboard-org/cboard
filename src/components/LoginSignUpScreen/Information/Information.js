import React, { Fragment } from 'react';
import { FormattedMessage } from 'react-intl';
import Typography from 'material-ui/Typography';

import messages from './Information.messages';

const Information = () => (
  <Fragment>
    <Typography
      align="center"
      className="LoginSignUpScreen__heading"
      variant="display3"
    >
      <FormattedMessage {...messages.heading} />
    </Typography>
    <p>
      <FormattedMessage {...messages.text} />
    </p>
  </Fragment>
);

export default Information;
