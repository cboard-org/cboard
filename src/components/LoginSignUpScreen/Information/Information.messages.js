import { defineMessages } from 'react-intl';

export default defineMessages({
  heading: {
    id: 'cboard.components.LoginSignUpScreenInformation.heading',
    defaultMessage: 'Cboard'
  },
  text: {
    id: 'cboard.components.LoginSignUpScreenInformation.text',
    defaultMessage: 'Sign up to sync your settings!'
  }
});
