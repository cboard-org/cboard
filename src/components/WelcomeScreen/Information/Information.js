import React, { Fragment } from 'react';
import { FormattedMessage } from 'react-intl';
import Typography from '@material-ui/core/Typography';

import messages from './Information.messages';

const Information = () => (
  <Fragment>
    <Typography
      align="center"
      className="WelcomeScreen__heading"
      variant="display3"
    >
      <FormattedMessage {...messages.heading} />
    </Typography>
    <p>
      <FormattedMessage {...messages.text} />
    </p>
  </Fragment>
);

export default Information;
