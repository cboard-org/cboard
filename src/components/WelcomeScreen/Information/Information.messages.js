import { defineMessages } from 'react-intl';

export default defineMessages({
  heading: {
    id: 'cboard.components.WelcomeScreenInformation.heading',
    defaultMessage: 'Welcome to Cboard'
  },
  text: {
    id: 'cboard.components.WelcomeScreenInformation.text',
    defaultMessage:
      'Cboard helps users with speech and language impairments to communicate with symbols and text-to-speech.'
  }
});
