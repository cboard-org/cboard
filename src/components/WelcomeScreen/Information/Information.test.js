import React from 'react';
import { shallowMatchSnapshot } from '../../../common/test_utils';
import Information from './Information';

jest.mock('./Information.messages', () => {
  return {
    heading: {
      id: 'cboard.components.WelcomeScreenInformation.heading',
      defaultMessage: 'Welcome to Cboard'
    },
    text: {
      id: 'cboard.components.WelcomeScreenInformation.text',
      defaultMessage:
        'Cboard helps users with speech and language impairments to communicate with symbols and text-to-speech.'
    }
  };
});

describe('Information tests', () => {
  test('default renderer', () => {
    shallowMatchSnapshot(<Information />);
  });
});
