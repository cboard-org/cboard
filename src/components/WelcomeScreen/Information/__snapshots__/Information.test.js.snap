// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Information tests default renderer 1`] = `
<React.Fragment>
  <WithStyles(Typography)
    align="center"
    className="WelcomeScreen__heading"
    variant="h2"
  >
    <FormattedMessage
      defaultMessage="Welcome to Cboard"
      id="cboard.components.WelcomeScreenInformation.heading"
      values={Object {}}
    />
  </WithStyles(Typography)>
  <p>
    <FormattedMessage
      defaultMessage="Cboard helps users with speech and language impairments to communicate with symbols and text-to-speech."
      id="cboard.components.WelcomeScreenInformation.text"
      values={Object {}}
    />
  </p>
</React.Fragment>
`;
